xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/BitfinexAdapters.java

```diff
@@ -25,10 +25,44 @@
 
   private BitfinexAdapters() {}
 
+  /**
+   * Converts a {@link Currency} to Bitfinex Symbol
+   * @param currency
+   * @return 
+   */
+  public static String adaptCurrency(Currency currency) {
```

Comment:
this will break applications that already depend codes used on that exchange or that do mapping by them selfs

this is a wider problem and should not be solved case by case but globaly in the future:
https://github.com/knowm/XChange/issues/2720

Reply:
The method 'adaptCurrency' is new and introduced here and should not break anything.

#2720 will solve the wider problem, but its still worth it to fix Bitfinex now as it contains a bug and does not translate DSH into DASH (and vice versa).

Reply:
if an application expects that the library returns "DSH" code for this exchange then this will break it

Reply:
Same thing is done among all other exchanges, e.g: XBT of Kraken is translated into normal BTC currency. 
You are correct that it might break existing applications, but it's incorrect logic to assume DASH and DSH are 2 different currencies and should be fixed.

Reply:
its done only in a few places and its wrong in my opinion
unifying currency codes is a far bigger problem then you think and giving an illusion that this library solved it is dangerous
BCC on one exchange is Bitcoin Cash and at another its BitConnect for example <- this is one of many examples that a global solution mast solve

Reply:
Ok then I am removing this fix and keeping it on my repository. You can stay with the bug.

Reply:
@timmolter - please decide.

Reply:
It looks like the code corresponding to this conversation is gone so I cannot really see exactly what was being done.

Reply:
Ok then, I will create new pull request specific for DASH <--> DSH translations.